app/src/main/java/com/sun/makeupwindow/data/source/local/dao/ProductDAOImpl.kt

```diff
@@ -134,6 +134,34 @@ class ProductDaoImpl private constructor(
         return products
     }
 
+    override fun searchProductLoadMore(wordSearch: String, totalItem: Int?): List<Product> {
+        val products = mutableListOf<Product>()
+        val query =
+            "SELECT * FROM ${Product.TABLE_NAME} WHERE ${Product.NAME} LIKE '%${wordSearch}%' OR " +
+                    "${Product.DESCRIPTION} LIKE '%${wordSearch}%' LIMIT ${totalItem} , ${LIMIT}"
```

Comment:
```suggestion
            "SELECT * FROM ${Product.TABLE_NAME} 
              WHERE ${Product.NAME}
              LIKE '%${wordSearch}%' OR " +
                    ....
```
có thể để các key làm tiền tố cho dễ nhìn